docs/sources/panels/field-configuration-options.md

```diff
@@ -83,19 +79,19 @@ Field overrides allow you to change the settings for one field (column in tables
 1. Continue to add overrides to this field by clicking **Add override property**, or you can click **Add override** and select a different field to add overrides to.
 1. When finished, click **Save** to save all panel edits to the dashboard.
 
-## Filter options
+## Override matchers
```

Comment:
Same comment as above. "matcher" is not a correct word. People understand what "filters" are. We will need to explain "matcher" and it will still look wrong.